js/djtilesource.js

```diff
@@ -1,92 +1,134 @@
+/*jslint browser: true*/
+/*global OpenSeadragon, Drupal*/
+/**
+ * @file
+ * Defines the source for exposing Djatoka to OpenSeadragon.
+ */
 (function($) {
+  'use strict';
+
+  // Remove IIFTileSource, as it assumes Djatoka responses belong to it for some reason.
+  $.IIIFTileSource = undefined;
```

Comment:
Taking a quick looks... Looks like the "data" configuration bit?: https://openseadragon.github.io/docs/openseadragon.js.html#line12084

... As in, we have `identifier`, `width` and `height`. (less-than ideal implementation in their case)... But we could be more definitive, I think? Just seems kind of weird to nuke... Would at least expect a big `XXX` bit in the comment.